### PR TITLE
'成功开发BackTop，路由backtop'

### PR DESCRIPTION
**Pull request by 1409070209**
github.com/1409070209/antue

### CODE REVIEW

File: components/backtop/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import BackTop from './back-top.vue'
+
```
Comment (sky-admin) [inline components/backtop/index.js]: where is your style file?

File: components/backtop/back-top.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="" @click="click">
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+  const BackTop = {
```
Comment (sky-admin) [inline components/backtop/back-top.vue]: use export default like other components

File: components/backtop/back-top.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="" @click="click">
```
Comment (sky-admin) [inline components/backtop/back-top.vue]: class = "" ??

Comment (sky-admin) [reply components/backtop/back-top.vue]: click handler method name is not good.

File: components/backtop/back-top.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        type: Number,
+        default: 0
+      },
+      callBack: {
+        type: Function,
+        default: () => {}
+      }
+    },
+    methods: {
+      click: function (e) {
```
Comment (sky-admin) [inline components/backtop/back-top.vue]: use `click (e) {...}`

File: examples/routers/index.js [hunk 5]
```diff
@@ -21,6 +21,7 @@ import Popover from './popover'
 import Popconfirm from './popconfirm'
 import Timeline from './timeline'
 import Dropdown from './dropdown'
+import Other from './other'
```
Comment (sky-admin) [inline examples/routers/index.js]: why named it 'other'?

File: components/backtop/index.js (hunk 1 above)
Comment (1409070209) [reply components/backtop/index.js]: style scoped

File: components/backtop/back-top.vue (hunk 2 above)
Comment (1409070209) [reply components/backtop/back-top.vue]: ojbk

File: examples/routers/index.js (hunk 5 above)
Comment (1409070209) [reply examples/routers/index.js]: in antd .it is other.backTop
https://ant.design/docs/react/introduce-cn

Comment (sky-admin) [reply examples/routers/index.js]: read it again.

File: components/backtop/index.js (hunk 1 above)
Comment (sky-admin) [reply components/backtop/index.js]: so what if antd upgrade style? modify .vue file is not a good idea when just upgrade style.

**Pull request closed without merging**